controllers/operands/deploymentHandler.go

```diff
@@ -43,27 +43,41 @@ func (h deploymentHooks) getEmptyCr() client.Object {
 
 func (deploymentHooks) justBeforeComplete(_ *common.HcoRequest) { /* no implementation */ }
 
-func (h deploymentHooks) updateCr(req *common.HcoRequest, Client client.Client, exists runtime.Object, _ runtime.Object) (bool, bool, error) {
-	found, ok := exists.(*appsv1.Deployment)
+func (h deploymentHooks) updateCr(req *common.HcoRequest, Client client.Client, exists runtime.Object, required runtime.Object) (bool, bool, error) {
+	deployment, ok1 := required.(*appsv1.Deployment)
```

Comment:
can you please use a different name to keep more evident that this is the required object?